core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly

```diff
@@ -21,26 +21,24 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
-
 <?jelly escape-by-default='true'?>
-
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:x="jelly:xml" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:x="jelly:xml">
     <j:new var="h" className="hudson.Functions" />
-    <st:statusCode value="503"/><!-- SERVICE NOT AVAILABLE -->
-    <st:setHeader name="Expires" value="0"/>
-    <st:setHeader name="Cache-Control" value="no-cache,no-store,must-revalidate"/>
+    <st:statusCode value="503" /><!-- SERVICE NOT AVAILABLE -->
+    <st:header name="Expires" value="0" />
+    <st:header name="Cache-Control" value="no-cache,no-store,must-revalidate" />
     <!-- response contentType header -->
-    <st:contentType value="text/html;charset=UTF-8"/>
+    <st:contentType value="text/html;charset=UTF-8" />
```

Comment:
Same for this page; it should use `<l:view>` (not sure how this one got missed back when that tag was introduced).